### PR TITLE
Add support for older cciss devices

### PR DESCRIPTION
**Pull request by xarses**
add detection for if devname is cciss and insert seperator if necessary

### CODE REVIEW

**Comment by rmoe** [comment]
I don't have HP hardware to test the cciss changes on. I did test both a CentOS and Ubuntu deployment on KVM with these changes and didn't see any issues.

**Comment by aglarendil** [comment]
Vladimir, please clarify. It looks like we do already have similar code in master, don't we ?

**Comment by kozhukalov** [comment]
No, we don't. I sent pull request with the same functionality a couple of weeks ago but then removed it due to duplication. Closed one is https://github.com/Mirantis/fuel/pull/762

File: deployment/puppet/cobbler/templates/scripts/pmanager.py [hunk 1]
```diff
@@ -22,6 +22,12 @@ def __init__(self, data):
         self._rcount = 0
         self._pvcount = 0
 
+    def _pseparator(self, devname):
+        pseparator=""
+        if devname.find("cciss")>0:
```
Comment (kozhukalov) [inline deployment/puppet/cobbler/templates/scripts/pmanager.py]: Please use space symbols according to pep8. I know that this whole script does not satisfy pep8 requirements but it looks awful without space symbols :-)

**Comment by kozhukalov** [comment]
+1

**Comment by aglarendil** [comment]
was it tested for any regressions?

**Comment by xarses** [comment]
we've been running with this patch applied during the ceph osd changes with out issue.

**Comment by angdraug** [comment]
Whitespace added, double quotes changed to single.

**Comment by angdraug** [comment]
US team had this fix included in our ISO images as soon as this pull request was created, and never seen any related breakage during 2 weeks we were testing it. It is also confirmed to work for the user who originally reported this bug.

**Comment by angdraug** [comment]
Fixed merge conflict with #773.

**Comment by aglarendil** [comment]
+2

**Pull request merged**